src/mantine-demos/src/demos/modals/context.tsx

```diff
@@ -25,8 +27,10 @@ function Demo() {
   const openContextModal = () =>
     modals.openContextModal('demonstration', {
       title: 'Test modal from context',
-      modalBody:
-        'This modal was defined in ModalsProvider, you can open it anywhere in you app with useModals hook',
+      customProps: {
```

Comment:
Let's rename customProps, it feels confusing, `innerProps` sounds a little better, I think.

Reply:
Done.